docker_tasks/vector_ingest/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM --platform=linux/amd64 osgeo/gdal:alpine-normal-latest
```

Comment:
@ranchodeluxe we're using Gdal 3.7 so we're able to use the "upsert" instead of "overwrite" if we choose to.